src/connection/mod.rs

```diff
@@ -50,9 +50,56 @@ fn test_list_function() {
 /// println!("{:?}", connection);
 /// ```
 pub fn create(s: &str, i: Interface, sc: Security, p: &str) -> Result<Connection, String> {
-    // Create a connection
-    // Get the connection
-    // Return the connection
+    let mut settings = std::collections::HashMap::new();
+
+    settings.insert("connection",
+                std::collections::HashMap::<&str, dbus::arg::Variant<dbus::MessageItem>>::new());
+    settings.insert("802-11-wireless",
+                std::collections::HashMap::<&str, dbus::arg::Variant<dbus::MessageItem>>::new());
+    settings.insert("802-11-wireless-security",
+                std::collections::HashMap::<&str, dbus::arg::Variant<dbus::MessageItem>>::new());
+
+    settings.get_mut("connection")
+        .unwrap()
```

Comment:
It'd probably be worth handling whatever errors we can properly, even if that is just printing a nice error message and exiting